src/main.py

```diff
@@ -408,9 +427,24 @@ def main():
     start_handler = CommandHandler('start', start)
     updater.dispatcher.add_handler(start_handler)
 
-    mensa_handler = CommandHandler('mensa', mensa)
+    mensa_handler = CommandHandler('mensa', alte_mensa)
```

Comment:
Now that the `alte_mensa` command is there, does it really make sense to still include the normal `mensa` command? Imo it's some sort of a duplicate, maybe even ambiguous.